src/main/java/frc/robot/controls/Driver.java

```diff
@@ -41,4 +46,48 @@ public Driver(Robot robot)
 	// METHODS //
 
 	// Add methods here which return values for various robot controls by reading the controllers.
+
+    /* =====================================
+    getThrottle() will return the current
+    value of the Y-Axis of the left stick on the controller
+    It will also handle adjusting
+    the reading according to the deadzone,
+    and according to the max throttle speed.
+    ===================================== */
+    public double getThrottle()
+    {
+        double throttleValue = 0;
+        
+        throttleValue = -dualshock.getY();
+        throttleValue *= Constants.MAX_THROTTLE_SPEED;
```

Comment:
set the `throttleValue` directly to `-dualshock.getY() * Constants.MAX_THROTTLE_SPEED` in one line instead of 2 for clarity.